curriculum/js0/6.js

```diff
@@ -7,7 +7,10 @@
  */
 
 const solution = (a, b) => {
-  return true;
+  if (a > b) {
+    return true;
+  }
+    return false;
```

Comment:
Please reduce the indentation on line 13, so that the second return statement is aligned with the start of the `if` statement on line 10.